itl/plugins-contrib.d/logfiles.conf

```diff
@@ -0,0 +1,115 @@
+/*
+ * Icinga2 CheckCommand definitions to monitor log files using check_logfiles.
+ * The check_logfiles is a Plugin for Icinga which scans log files for specific patterns.
+ * Available from here: https://labs.consol.de/nagios/check_logfiles/
+ */
+
+object CheckCommand "log" {
```

Comment:
Please use the full plugin name without the `check_` prefix. `log` is too generic, and also reserved for internal use.

Reply:
Besides, the documentation and parameters also refer to `logfiles` as check command name.